### ISSUE TITLE
Alternative Routes

### ISSUE THREAD

**Issue by zijiazhai** [mapbox/mapbox-navigation-ios#687]
hi, guys, I am using 'MapboxNavigation', '~> 0.8.0'.

And my question is that is there a way to show users alternative routes, and let them choose?

options.includesAlternativeRoutes = true
 for route_ in routes!{
                self?.mapView.showRoute(route_)
            }

even though routes.count == 2, there is still only one route in mapView.

**Comment by zijiazhai** [comment]
hi @bsudekum can we add a new feature to Avoid Toll/ Avoid Highway etc later.  thanks

**Comment by 1ec5** [comment]
Alternative route selection was implemented in #789.

> can we add a new feature to Avoid Toll/ Avoid Highway etc later

This is probably worth a separate issue. It’s blocked by mapbox/MapboxDirections.swift#180 anyhow.
